WalletWasabi/Blockchain/Analysis/FeesEstimation/IThirdPartyFeeProvider.cs

```diff
@@ -6,4 +6,7 @@ public interface IThirdPartyFeeProvider
 
 	AllFeeEstimate? LastAllFeeEstimate { get; }
 	bool InError { get; }
+	bool IsPaused { get; set; }
+
+	public void TriggerOutOfOrderUpdate();
```

Comment:
TriggerOutOfOrderUpdate => TriggerUpdate ? TriggerInstantUpdate

Reply:
Changed to TriggerUpdate